frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -56,66 +60,73 @@ export const SecurityBreakdownPopup: React.FC<WatchK8sResults<WatchImageVuln>> =
+                        <div>
+                          <ExclamationTriangleIcon
+                            color={priority.color.value}
+                            alt={priority.title}
+                          />
+                          &nbsp;
+                          {_.uniqBy(vulnsFor(priority.value), 'metadata.name').length}{' '}
+                          {priority.title}
+                        </div>
+                      </div>
+                    ) : null,
+                  )
                   .toArray()}
-                title={`${resource.length} total`}
-              />
+              </div>
+              <div>
+                <Link
+                  to={`/k8s/all-namespaces/${referenceForModel(ImageManifestVulnModel)}`}
+                  aria-label="View all"
+                >
+                  <ChartDonut
+                    colorScale={vulnPriority.map((priority) => priority.color.value).toArray()}
+                    data={vulnPriority
+                      .map((priority) => ({
+                        label: priority.title,
+                        x: priority.value,
+                        y: _.uniqBy(vulnsFor(priority.value), 'metadata.name').length,
+                      }))
+                      .toArray()}
+                    title={`${_.uniqBy(resource, 'metadata.name').length} total`}
+                  />
+                </Link>
+              </div>
             </div>
           </div>
           {!_.isEmpty(fixableVulns) && (
-            <>
+            <div className="co-status-popup__section">
               <div className="co-status-popup__row">
-                <div className="co-status-popup__text--bold">Fixable Vulnerabilities</div>
+                <div>
+                  <span className="co-status-popup__text--bold">Fixable Container Images</span>
+                  <span className="text-secondary">&nbsp;({fixableVulns.size} total)</span>
+                </div>
               </div>
-              {_.take([...fixableVulns.values()], 5).map((v) => (
+              <div className="co-status-popup__row">
+                <span className="co-status-popup__text--bold">Impact</span>
+                <span className="co-status-popup__text--bold">Vulnerabilities</span>
+              </div>
+              {_.sortBy(_.take([...fixableVulns.values()], 5), [
+                (v) => priorityFor(v.status.highestSeverity).index,
+              ]).map((v) => (
                 <div className="co-status-popup__row" key={v.metadata.name}>
                   <span>
-                    <SecurityIcon
-                      color={
-                        vulnPriority.find((p) => p.title === _.get(v.status, 'highestSeverity'))
-                          .color.value
-                      }
+                    <ExclamationTriangleIcon
+                      color={priorityFor(_.get(v.status, 'highestSeverity')).color.value}
```

Comment:
We've been trying to remove `_.get` in favor of optional chaining when already updating the code.

```suggestion
                      color={priorityFor(v.status?.highestSeverity).color.value}
```

Having said that, I'm not sure `_.get` even helps here since we'll have a runtime error on lin 142 if `v.status` is `undefined`.